docs/_guide/templates.md

```diff
@@ -375,21 +386,22 @@ You can compose LitElement templates from other LitElement templates. In the fol
 
 ```js
 class MyPage extends LitElement {
+  ...
   render() {
     return html`
-      ${this.headerTemplate}
-      ${this.articleTemplate}
+      ${this.headerTemplate(this.article.title)}
```

Comment:
Instead of passing an argument I would just access `this.article.title` in the headerTemplate. Or keep the argument and make header template standalone function to show that they don't need to be class methods.

Reply:
Urgh. I was trying to show functions here because someone was confused by the getters. 

Are standalone methods too confusing? I like showing that they don't need to be class methods, so I'll go with that.